new-lamassu-admin/src/pages/Triggers/Wizard.js

```diff
@@ -236,21 +272,28 @@ const Wizard = ({ onClose, save, error, currency }) => {
         />
         <Formik
           validateOnBlur={false}
-          validateOnChange={false}
+          validateOnChange={true}
```

Comment:
Maybe this should remain false?

Reply:
Having this on false, would make the form much less responsive to user interaction

Reply:
yeah but I once worked on a card where I was supposed to force the forms validation only after submit. Check this with Rafael if you can.